core/pythonAction/pythonrunner.py

```diff
@@ -55,7 +55,7 @@ def build(self, message):
 
         try:
             filename = os.path.basename(self.source)
-            self.fn = compile(code, filename=filename, mode='exec')
+            self.fn = compile(code, filename=filename, mode='exec')            
```

Comment:
Can you remove the extra whitespace at the end of this line?

Reply:
oh! thx

Reply:
> Can you remove the extra whitespace at the end of this line?

hi , I resubmitted and modified some test cases. Please review again, thank you very much.